test-libz-rs-sys/src/inflate.rs

```diff
@@ -1205,6 +1205,11 @@ fn gzip_chunked_2_bytes() {
     gzip_chunked(2);
 }
 
+#[test]
+fn gzip_chunked_16_bytes() {
+    gzip_chunked(16);
+}
```

Comment:
Optional: you may consider throwing on in some non-powers-of-two checks as well, since there's no guarantee (AFAICT) a user will choose chunks that are powers of two. Or could throw `gzip_chunked()` into a `quickcheck`. 

(Then again, it looks like we do fuzz chunked inflate, so maybe it's not necessary.)

Reply:
added some extra tests, but they are all good. 

We do fuzz chunked inflate, but not actually in combination with gzip. In general I suspect gzip headers are under-tested at the moment.